main.star

```diff
@@ -21,6 +21,7 @@ beacon_metrics_gazer = import_module(
     "./src/beacon_metrics_gazer/beacon_metrics_gazer_launcher.star"
 )
 dora = import_module("./src/dora/dora_launcher.star")
+blobscan = import_module("./src/blobscan/blobscan_launcher.star")
```

Comment:
@PabloCastellano can you add a testcase in `.github/tests` that spins up blobscan! there is a tools example; it might be as simple as adding a line there

Reply:
https://github.com/kurtosis-tech/ethereum-package/blob/main/.github/tests/mix-with-tools.yaml - just enrich this one;